components/05-atoms/media/avatar/_avatar.scss

```diff
@@ -2,6 +2,7 @@ $avatar-size-xs: ms(10); //~40px
 $avatar-size-sm: ms(14); //~55px
 $avatar-size-md: ms(18); //~80px
 $avatar-size-base: ms(22); //~120px
+$avatar-size-xl: $avatar-size-base*1.95; //r = ~234 px
```

Comment:
Wert bitte auch in der ms Schreibweise ausdrücken wie bei den anderen Variablen auch.